network/edge-services/how-to/configure-cache.mdx

```diff
@@ -88,9 +88,19 @@ This allows you to specify the precise objects that you want to clear from the c
 
     The specified objects are purged from your cache and you are returned to the Edge Services dashboard.
 
+## How to use Cache-Control
+
+To avoid repeated purging, you can also set the `Cache-Control` HTTP header directly on your files/objects. This allows you to define how long items should remain in the cache before being freshly retrieved from the origin. Edge Services will respect this header when determining from where to fetch content.
```

Comment:
```suggestion
To avoid repeated purging, you can also set the `Cache-Control` HTTP header directly on your files/objects. This allows you to define how long items should remain in the cache before being freshly retrieved from the origin. Edge Services will respect this header when determining where to fetch content.
```